src/resources/js/components/EditPresentation.vue

```diff
@@ -0,0 +1,90 @@
+<template>
+    <v-form
+        ref="form"
+        v-model="valid"
+        lazy-validation
+    >
+      <v-text-field
+            v-model="updatePresentationInfo.title"
+            :rules="titleRules"
+            label="Titre"
+      />
+
+      <v-datetime-picker
+        label="Date et Heure"
+        v-model="updatePresentationInfo.date"
+        clearText="annuler"
+      >
+        <template slot="dateIcon">
+          <v-icon>mdi-calendar</v-icon>
+        </template>
+        <template slot="timeIcon">
+          <v-icon>mdi-clock</v-icon>
+        </template>
+      </v-datetime-picker>
+
+      <v-btn
+            :disabled="!valid"
+            color="success"
+            class="mr-4"
+            @click="submitChange"
+            >
+            Appliquer
+      </v-btn>
+    </v-form>
+</template>
+
+<script>
+import dateFormat from 'dateformat';
+
+  let alert = {};
+  export default {
+    props: ['parentRefs'],
+    data() {
+      return {
+        valid: true,
+        titleRules: [
+          (v) => v === '' || (v && v.length <= 20) || 'Le titre doit contenir moins de 20 caractères',
+        ],
+        updatePresentationInfo: {
+          date: '' ,
+          title: '',
+       },
+      };
+    },
+    beforeMount() {    
+      alert = this.parentRefs.alert;
+      
+      window.axios
+        .get(`presentations/${this.$route.params.idPresentation}`)
+        .then((response) =>{ 
+          this.updatePresentationInfo.title = response.data.title;
+          this.updatePresentationInfo.date = dateFormat(response.data.date, 'yyyy-mm-dd HH:MM');
+        })
+        .catch(() => {
+          alert.showMessage('error', 'Une erreur est survenue lors du traitement de votre demande');
+        })
+    },
+    methods: {
+      submitChange() {
+        const data = this.updatePresentationInfo;
```

Comment:
unused variable should be removed,
also you should verify your linter as it is supposed to throw an error if you have unused variables

Reply:
Changed in c34013a